src/main/java/core/basesyntax/storage/Storage.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.storage;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage implements FruitStorage {
+    private static final Map<String, Integer> fruits = new HashMap<>();
```

Comment:
The use of a static map means that the storage is shared across all instances of `Storage`. Ensure this design choice aligns with the application's requirements, as it may lead to unexpected behavior if multiple instances are expected to have independent storage.